code/src/pages/not-found/messages/en.json

```diff
@@ -0,0 +1,5 @@
+{
+  "pageNotFound": "Page Not Found",
+  "notFoundDesc": "The page you are looking for might have been removed, had its name changed, or is temporarily unavailable",
```

Comment:
Desc its short name of descending. Name it description

Reply:
> Desc its short name of descending. Name it description

Is it important for key naming ? I just used short convention

Reply:
In this case, it will be more informative. Just add a little bit more letters and our lives will become easier :)

Reply:
> In this case, it will be more informative. Just add a little bit more letters and our lives will become easier :)

The such case we also used in `s2s` and everything was ok. To use short naming why not

Reply:
It's not `s2s`, so try to follow best practices

Reply:
> It's not `s2s`, so try to follow best practices

Sure. Show me where it's has written in our rules. I'll rewrite it with best practices :D
For now It's your suggestion not more. Are u agree ??